dashboard/app/controllers/api/v1/census/census_controller.rb

```diff
@@ -84,6 +91,13 @@ def create
     CHECKBOX_FIELDS.map do |field|
       census_params[field] = false unless census_params[field]
     end
+
+    # The database cannot handle utf8mb4 characters.
+    # Strip them out before saving.
+    FREE_TEXT_FIELDS.map do |field|
+      census_params[field] = census_params[field].strip_utf8mb4 unless census_params[field].nil?
+    end
```

Comment:
nit: use `each` instead of `map` since the returned enumerable is not consumed. Same on line 91

Reply:
Changed.